azext_iot/central/providers/device_provider.py

```diff
@@ -221,3 +221,11 @@ def _dps_populate_essential_info(self, dps_info, device_status: DeviceStatus):
             "error": error.get(device_status),
         }
         return filtered_dps_info
+
+    def get_device_registration_summary(self, central_dns_suffix="azureiotcentral.com"):
```

Comment:
>get_device_registration_summary [](start = 8, length = 31)

as we discussed offline maybe drop a logger.warn("This command may take a long time to complete if your app contains a lot of devices") or similar